packages/ariakit/src/menu/__examples__/menu-item-radio/style.css

```
@@ -0,0 +1,2 @@
+@import "../../../button/__examples__/button/style.css";
+@import "../menu/style.css";
```

I think menu already imports button styles.
```suggestion
@import "../menu/style.css";
```